Colours_And_Fonts_style_v1.css

```diff
@@ -240,10 +346,11 @@ img {
 
 .footer {
 	grid-column-end: span 2;
-	background-color: #B3E8FF;
+	background-color: #FBCA01;

```

Comment:
I changed the colour of the footer to yellow and not blue as it looks better because I made the box4 above the footer navy blue so I can add links on it.